config/formats.js

```diff
@@ -1891,16 +1891,17 @@ exports.Formats = [
 	{
 		name: "[Gen 7] Doubles Monotype",
 		desc: [
-			"All the Pok&eacute;mon on a team must share a type. The battle is fought in the Doubles Format. Banlist is the same as Doubles OU. Requested by Ashley the Pikachu. \"Coded\" by Gligars.",
+			"All the Pok&eacute;mon on a team must share a type. The battle is fought in the Doubles Format. Banlist is the same as Doubles OU. Requested by Ashley the Pikachu. \"Coded\" by Lycanium Z.",
```

Comment:
Shouldnt that change be done on main?

Reply:
If you mean main server, Probably not, Since i have to go through a long and grueling process of getting it accepted on to main through the monotype community

Reply:
so no